normandy/control/static/control/js/components/ActionForm.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { reduxForm } from 'redux-form'
+import { _ } from 'underscore'
+import HeartbeatForm from './action_forms/HeartbeatForm.jsx'
+import ConsoleLogForm from './action_forms/ConsoleLogForm.jsx'
+
+export class ActionForm extends React.Component {
+  render() {
+    const { fields, arguments_schema, name } = this.props;
+
+    let childForm = 'No action form available';
+
+    switch(name) {
+      case 'show-heartbeat':
+        childForm = (<HeartbeatForm fields={fields} />);
+        break;
+      case 'console-log':
+        childForm = (<ConsoleLogForm fields={fields} />);
+        break;
+    }
+
+    return (
+      <div id="action-configuration">
+        <i className="fa fa-caret-up fa-lg"></i>
+        <div className="row">
+          <p className="help fluid-4">{arguments_schema.description || arguments_schema.title }</p>
```

Comment:
If we move this to the child form and make each form include a description itself instead of pulling it from the schema we can remove dependence on the schema entirely, which would be nice.